projects/Mallard/src/helpers/weather.ts

```diff
@@ -56,9 +56,12 @@ const fetchFromWeatherApi = async <T>(path: string): Promise<T> => {
 
 const getIpBasedLocation = async () => {
     const ip = await getIpAddress()
-    const accuLoc = await fetchFromWeatherApi<AccuWeatherLocation>(
+    const accuLoc = await fetchFromWeatherApi<AccuWeatherLocation | null>(
         `locations/v1/cities/ipAddress?q=${ip}&details=false`,
     )
+    if (accuLoc == null) {
+        throw new CannotFetchError('Couldn\t identify location from latlong')
```

Comment:
Do we want to throw? Will this then get picked up in Sentry as an error we dont want?

Reply:
No, this is caught by a `try..catch` later in the file, specifically targeting `CannotFetchError`, and the weather will simply not show.